atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -185,11 +212,30 @@ def _render_mail(self, rebuild, success, auto_canceled, manual_canceled):
             'user': '<autorebuild>' if rebuild else self.submitter,
             'logs': url
         }
-        return (subject_template % formatting_dict, body_template % formatting_dict)
 
-    def _send_mail(self, receivers_list, subject, body):
-        """Actually sends the mail with `subject` and `body` to all members of `receivers_list`."""
-        msg = MIMEText(body)
+        log_files = None
+        if rebuild and endstate == 'Failed':
+            log_files = self._fetch_log_files()
+
+        return (subject_template % formatting_dict, body_template % formatting_dict, log_files)
+
+    def _send_mail(self, receivers_list, subject, body, log_files=None):
+        """Actually sends the mail with `subject` and `body` and optionanl log_file attachements
+        to all members of `receivers_list`."""
+        if log_files:
+            msg = MIMEMultipart()
+            msg.attach(MIMEText(body))
```

Comment:
We should probably use a different body message. Otherwise, it'll still contain link to Koji task. This can be handled by another story/PR.